.changeset/yellow-students-tie.md

```diff
@@ -0,0 +1,20 @@
+---
+"@finos/babel-preset-legend-studio": patch
+"@finos/eslint-plugin-legend-studio": patch
+"@finos/legend-studio": patch
+"@finos/legend-studio-app": patch
+"@finos/legend-studio-components": patch
+"@finos/legend-studio-dev-utils": patch
+"@finos/legend-studio-manual-tests": patch
+"@finos/legend-studio-network": patch
+"@finos/legend-studio-plugin-management": patch
+"@finos/legend-studio-plugin-query-builder": patch
+"@finos/legend-studio-plugin-tracer-zipkin": patch
+"@finos/legend-studio-preset-dsl-text": patch
+"@finos/legend-studio-preset-dummy": patch
+"@finos/legend-studio-preset-external-format-json-schema": patch
+"@finos/legend-studio-shared": patch
+"@finos/stylelint-config-legend-studio": patch
+---
+
+Add support for BigQuery
```

Comment:
Given how urgent you need this, we will merge this, but please prune the changeset next time.

Reply:
We would have 2 changets: 1 for your change `Add support for BigQuery` for only `@finos/legend-studio`, 1 for all the changes across modules